vhd_format_lwt/jbuild

```diff
@@ -0,0 +1,16 @@
+(library
+ ((name vhd_format_lwt)
+  (public_name vhd-format-lwt)
+  (libraries
+   (cstruct.lwt
+    lwt
+    lwt.preemptive
+    lwt.unix
+    mirage-block
+    mirage-types-lwt
+    oUnit
+    threads
```

Comment:
Does this means that we have to add an OPAM dependency on `base-threads`?

Reply:
I think that's what other packages are moving toward as well, but I may be wrong

Reply:
It's not needed anyway, it compiles without it, I'll remove this dependency.